_health-care/_js/utils/options-for-select.js

```diff
@@ -237,6 +237,18 @@ const maritalStatuses = [
   'Divorced'
 ];
 
+const branchesServed = [
+  'Army',
+  'Airforce',
+  'Marine',
+  'Navy'
+];
+
+const dischargeTypes = [
```

Comment:
Sorry I missed this question. The point of the XML above is to have a full text description for the Vet and the abbreviated versions for the XML form/PDF form.